roles/apache/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+apache:
+  modules:
+    - libapache2-mod-wsgi
+    - libapache2-mod-uwsgi
+  listen: []
+  logs:
+    - paths:
+        - /var/log/apache2/access.log
+      fields:
+        # type: <custom type defined in receiving Logstash>
+        tags: apache_access
+    - paths:
+        - /var/log/apache2/error.log
+      fields:
+        # type: <custom type defined in receiving Logstash>
+        tags: apache_error
+  ssl:
+    settings: |
+      SSLEngine on
```

Comment:
These are settings you could use in a task if you wanted to use SSL  with apache. Horizon and keystone both are not using SSL for this PR.